### ISSUE TITLE
No delete customer unless 'No Reservation' filter was used

### ISSUE THREAD

**Issue by jevers76** [dolson334/banterCustomer#5]
Customers without reservation found with a search not using the  'No Reservation' filter will not show the delete customer

**Comment by dolson334** [comment]
@jevers76 yes this is true, you have to be on the no reservation filter. We will add a ticket for this. Since there is an easy work around Im not sure what the version will be yet.
